src/System.Text.Json/tests/Serialization/ExtensionDataTests.cs

```diff
@@ -178,23 +178,54 @@ public static void IgnoredDataShouldNotBeExtensionData()
         }
 
         [Fact]
-        public static void InvalidExtensionValue()
+        public static void ExtensionPropertyObjectValue_Empty()
+        {
+            // Baseline
+            ClassWithExtensionPropertyAlreadyInstantiated obj = JsonSerializer.Parse<ClassWithExtensionPropertyAlreadyInstantiated>(@"{}");
+            Assert.Equal(@"{""MyOverflow"":{}}", JsonSerializer.ToString(obj));
+        }
+
+        [Fact]
+        public static void ExtensionPropertyObjectValue()
```

Comment:
Can you also add a simple test case that matches the sample from the original issue: https://github.com/dotnet/corefx/issues/38238#issue-452258055

Reply:
done